subcommittees/proposal_instructions.md

```diff
@@ -0,0 +1,35 @@
+##Software Carpentry Subcommittee & Task Force Program
+
+One of the greatest things about Software Carpentry is the huge diversity of people and ideas that have come together to make this project what it is: a worldwide collaboration of folks working together to build software skills. Your contributions to the community have made Software Carpentry a huge success - but we can go even further. To meet the needs of the growing global community, the 2016 Steering Committee is creating the opportunity for you to make Software Carpentry what your community needs it to be, via our **newly revamped Subcommittee & Task Force Program**.
+
+Do you have an idea for an activity or service that would make Software Carpentry even more awesome to participate in for its instructors or students? Is there a project, tool or document that would benefit you and your community? If so, **we want you to submit a proposal for a Subcommittee or Task Force**; here’s how:
+
+### Step 1: Subcommittee or Task Force?
+The first thing to decide, is whether you’d like to propose a Subcommittee or a Task Force.
+**Subcommittees** pursue ongoing activities, services or tasks that enrich the community. For example, the Mentorship Committee hosted Instructor debriefings every couple of weeks in 2015 to help instructors talk about their experiences and connect with each other.
+**Task Forces** get together to tackle a specific, one-time goal, like hosting an event, producing a document, or studying a question relevant to the community. For example, a few people got together in late 2015 to plan and deliver our first ever [Instructor and Helper Retreat](http://swcarpentry.github.io/instructor-retreat-2015/).
+
+### Step 2: Draft a Plan
+The Steering Committee wants you to be successful! And that means getting started with a really clear plan of what you want to achieve. Please draft a proposal that addresses the following:
+
+ - What are the **specific goals** you want to achieve?
+ - What **steps** will you take to achieve those goals?
+ - What is your general **timeline**? 
+ - How will you periodically **measure** your progress and success?
+ - What **community members** (if any) will to help you achieve your goals?
+
+Remember, a modest **budget will be available**, approved on an item-by-item basis by the Steering Committee, to help you realize the goals of your Subcommittee or Task Force, once approved, so don’t forget to factor that into your plan.
```

Comment:
I think the “… item-by-item…” portion of this sentence will let us drop the “once approved,”